mappings/UofM.py

```diff
@@ -75,6 +70,49 @@ def formatIdentifiers(self):
                     return newISBNList
                 
         return self.record.identifiers
+    
+    def formatSubjects(self):
+        subjectList = []
+
+        if '|' in self.record.subjects:
+            subjectList = self.record.subjects.split('|')
+            newSubjectList = [f'{subject}||' for subject in subjectList] 
+            return newSubjectList
+        else:
+            subjectList.append(f'{self.record.subjects}||')
+            return subjectList
+    
+    def formatRights(self):
+        # The pipe delimiter is to separate the Rights table attributes into this format:
+        # source|license|reason|statement|date 
+        # which makes it easy to place the right data into the columns when clustered
+
+        rightsElements = self.record.rights.split('|') if self.record.rights else [''] * 5
```

Comment:
I'd say if the rights are None, let's just return None. so we can have 

```
if not self.record.rights: return None

rightsElements = self.record.rights.split('|')
```

Reply:
Sure. I can do this and I'll delete the if conditional in the `applyFormatting `method:

```
if self.record.rights:
            self.record.rights = self.formatRights()
```